dependencies/pom.xml

```diff
@@ -1445,6 +1445,13 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
+                <groupId>io.grpc</groupId>
+                <artifactId>grpc-bom</artifactId>
+                <version>${version.lib.grpc}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
```

Comment:
In addition to importing the grpc-bom you should remove all of the `<groupId>io.grpc</groupId>` dependencies higher up in the `dependencyManagement` section since the bom replaces the need to manage each `io.grpc` artifact version individually.